docs/site_organization/organizing_the_site.md

```diff
@@ -72,6 +72,21 @@ to have relations only to Content items of a specific, or any Content Type.
     You can delete a Content Type only when there are no Content items that belong to it.
     This also includes Content items in the Trash.
 
+### Setting relation starting location
+
+[Relation Field types](../creating_content_basic.md) allow to configure the location in the Content tree which opens in the Universal Discovery Widget (UDW).
+
+To set the starting location for the relation, perform the following steps. 
+Go to **Content Types** and edit the respective Content Type.
+Next, go to **Field definitions**. In the `objectrelation` Field, select from the available options:
+
+- **Default** - the starting location is automatically assigned to the default location in the tree of a created Content.
+- **Browse** - use to manually select the location from the Universal Discovery Widget.
+- **Content location** - the starting location is the location of the currently edited Content. For example, if you edit the Content item with the location `50`, it sets the starting location to this value with children under this location.
```

Comment:
```suggestion
- **Content location** - the starting location is the location of the Content item that is edited by the user. For example, if the user edits the Content item with the location `50`, it sets the starting location to this value with children under this location.
```

"currently edited Content" - item or type? Is the content type currently edited by the administrator user, or a content item currently edited by the end user?